horizomer/utils/tests/test_tree.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The Horizomer Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join, dirname, realpath
+from skbio import TreeNode
+
+from horizomer.utils.tree import (compare_topology,
+                                  read_taxdump,
+                                  build_taxdump_tree)
+
+
+class TreeTests(TestCase):
+
+    def setUp(self):
+        """ Set up working directory and test files
+        """
+        # test output can be written to this directory
+        self.working_dir = mkdtemp()
+
+        # test data directory
+        datadir = join(dirname(realpath(__file__)), 'data')
+
+        # test data files
+        self.nodes_fp = join(datadir, 'nodes.dmp')
+        self.names_fp = join(datadir, 'names.dmp')
+
+    def tearDown(self):
+        # there isn't any file to remove at the moment
+        # but in the future there will be
+        rmtree(self.working_dir)
+
+    def test_compare_topology(self):
+        """Test comparing topologies of two trees."""
+        # test identical Newick strings
+        tree1 = '(a,b)c;'
+        tree2 = '(a,b)c;'
+        obs = compare_topology(TreeNode.read([tree1]), TreeNode.read([tree2]))
```

Comment:
just out of curiosity: why do you pass tree1 within a list?

Reply:
Because that's how skbio reads in newick strings (here's a [unittest](https://github.com/biocore/scikit-bio/blob/master/skbio/tree/tests/test_tree.py#L56) for example).

Reply:
Right!

Reply:
I wasn't aware of this. Cool. I always took the detour via StringIO:
```TreeNode.read(StringIO('(a,b)c;'))```
And thus we see how useful code reviews are :-)

Reply:
👍